src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookConsumer.java

```diff
@@ -46,7 +46,7 @@ void process(RefsChangedWebhookEvent event) {
 
         try (ACLContext ctx = ACL.as(ACL.SYSTEM)) {
             BitbucketWebhookTriggerRequest request =
-                    BitbucketWebhookTriggerRequest.builder().actor(event.getActor()).build();
+                    BitbucketWebhookTriggerRequest.builder().actor(event.getActor().orElse(null)).build();
```

Comment:
Personally, I think this will read better as

```java
            BitbucketWebhookTriggerRequest.Builder requestBuilder = BitbucketWebhookTriggerRequest.builder();
            event.getActor().ifPresent(requestBuilder::actor);
```